test/cmd/policy.sh

```diff
@@ -34,4 +34,39 @@ os::cmd::expect_success_and_not_text 'oc get rolebinding/cluster-admin --no-head
 
 os::cmd::expect_success 'oc policy remove-group system:unauthenticated'
 os::cmd::expect_success 'oc policy remove-user system:no-user'
+
+# adjust the cluster-admin role to check defaulting and coverage checks
+# this is done here instead of an integration test because we need to make sure the actual yaml serializations work
+workingdir=$(mktemp -d --suffix=policy)
+cp ${OS_ROOT}/test/fixtures/bootstrappolicy/old_alternate_cluster_admin.yaml ${workingdir}
```

Comment:
better name for the fixture... is it the cluster-admin role pre-api-group? `cluster_admin_1.0.yaml`, `cluster_admin_without_apigroups.yaml`, will be helpful in the future